templates/tfengine/recipes/folder/project.yaml

```diff
@@ -23,6 +23,11 @@ templates:
         name: '"mock-folder"'
     INPUTS:
       folder_id: "dependency.parent_folder.outputs.name"
+  flatten: [
+    {{if has . "PROJECT_TERRAFORM"}}
+    { key: "PROJECT_TERRAFORM" }
```

Comment:
How do the single brackets work in this context?

Reply:
I was actually able to get around this because this is currently the only flatten happening for this template.

However, the reason is i had flatten: [ ]. This is the JSON mode and thus requires JSON style objects (e.g { }).